### DISCUSSION TITLE
[entgql] Generate multiple sources

### DISCUSSION THREAD

**Discussion by cliedeman** [99designs/gqlgen#1994] [Ideas]
EarlySourceInjector allow caters for injecting a single graphql file.
I would like to be able to generate multiple for easier debugging.

E.g.
relay.graphql, enums.graphql, connection1.graphql. etc.

I guess this could also be done as a new plugin to avoid breaking backwards compatibility
```go
type EarlySourcesInjector interface {
	InjectSourcesEarly() ([*ast.Source], error)
}
```

**Comment by cliedeman** [comment]
PR Created #2011

**Comment by StevenACoffman** [comment]
@cliedeman Hey, if you can make a PR to Entgql I would appreciate it!

**Reply by cliedeman** [reply]
Hey @StevenACoffman i went back to the repo and checked. And ent doesn't use inject sources. Or it previously used it when this change was first made and has since been removed.

**Reply by StevenACoffman** [reply]
Awww... bummer. Thanks for looking!
